taskcluster/ci/tests/kind.yml

```diff
@@ -72,6 +75,7 @@ tasks:
     run-on-tasks-for: ["github-push", "github-pull-request"]
 
   taskgraph-definition:
+      expires-after: "100 days"
```

Comment:
Is this necessary given the change to the default in `config.yml`?

Reply:
This value is defaulting to 28 days and a task can't expire before it's deadline.

That's the default for tasks that run on `"github-pull-request"`

https://github.com/taskcluster/taskgraph/blob/3cd7f06a4a1e3ef72e70d4063e1170bdd8fd74b4/src/taskgraph/transforms/task.py#L1078

https://github.com/taskcluster/taskgraph/blob/3cd7f06a4a1e3ef72e70d4063e1170bdd8fd74b4/src/taskgraph/parameters.py#L233

So it should not be an issue for the training tasks. But, I don't really want to set this on all the CI tasks :/

I could also override the deadline. But I got some schema error when I tried

Reply:
> This value is defaulting to 28 days and a task can't expire before it's deadline.
> 
> That's the default for tasks that run on `"github-pull-request"`
> 
> https://github.com/taskcluster/taskgraph/blob/3cd7f06a4a1e3ef72e70d4063e1170bdd8fd74b4/src/taskgraph/transforms/task.py#L1078
> 
> https://github.com/taskcluster/taskgraph/blob/3cd7f06a4a1e3ef72e70d4063e1170bdd8fd74b4/src/taskgraph/parameters.py#L233
> 
> So it should not be an issue for the training tasks. But, I don't really want to set this on all the CI tasks :/
> 
> I could also override the deadline. But I got some schema error when I tried

Ah, sorry! I was thinking about `deadline`, but this is about `expires`. 100 days is fine for that.

Reply:
I made a patch for the schema thing https://github.com/taskcluster/taskgraph/pull/344

I feel like overriding the deadline makes more sense for these, because this tasks should probably expire sooner.

Reply:
I made a patch for the schema thing https://github.com/taskcluster/taskgraph/pull/344

I feel like overriding the deadline makes more sense here. These tasks should probably expire sooner.